spot-client/src/spot-tv/ui/components/meeting-frame/meeting-frame.js

```diff
@@ -324,6 +357,52 @@ export class MeetingFrame extends React.Component {
         });
     }
 
+    /**
+     * Updates the current known display state of the filmstrip, whether it is
+     * displayed or hidden.
+     *
+     * @param {Object} event - The event returned from the external api about
+     * the filmstrip change.
+     * @param {boolean} visible - Whether or not the filmstrip strip is
+     * currently set to be displayed all the time.
+     * @private
+     * @returns {void}
+     */
+    _onFilmstripDisplayChanged({ visible }) {
+        this._isFilmstripVisible = visible;
+    }
+
+    /**
+     * Adds a participant from the list of known current participants in the
+     * meeting.
+     *
+     * @param {Object} event - An object with participant information.
+     * @param {string} event.id - The jitsi-meet ID of the participant.
+     * @param {string} even.displayName - The current name of the participant.
+     * @private
+     * @returns {void}
+     */
+    _onParticipantJoined({ id, displayName }) {
```

Comment:
I would probably prefer to have this kind of logic in a class separate from UI component. This way it makes it less likely for someone else to duplicate this same logic again, but because it's just the two of us right now it's not a big deal.